.github/workflows/docker.yml

```diff
@@ -0,0 +1,50 @@
+name: "Push docker image to GCR"
+
+on:
+  push:
```

Comment:
is it OK to leave it on push to main or should we make releases for this even though its mostly an internally run tool and we may not neccessarily version it?

Reply:
In taiko-client, we will build a `main` tag image on pushes to main ( that `main` tag image will be used in alpha-2 testnet ) , and build a `latest` tag image when there is a new github release tag ( will be used in alpha-1 ) , since relayer is in a mono repo, not sure whether trigger building images based on new release tags can work well? I think its OK to leave it on push to main for now, and then try to figure out a better way to push new images based on release versions later on.